mypyc/test-data/run-strings.test

```diff
@@ -140,11 +140,21 @@ def test_partition() -> None:
     with assertRaises(ValueError, "empty separator"):
         rpartition(s_partition, "")
 
+def contains(s: str, o: str) -> bool:
+    return o in s
+
 def getitem(s: str, index: int) -> str:
     return s[index]
 
 s = "abc"
 
+def test_contains() -> None:
+    assert contains(s, "a") is True
+    assert contains(s, "abc") is True
```

Comment:
Ideas for a few further tests: 
* `assert contains(s, "bc") is True`
* `assert contains(s, "abcd") is False`
* `assert contains(s, "bb") is False`